### DISCUSSION TITLE
Anyway to switch indexer?

### DISCUSSION THREAD

**Discussion by papampi** [pymedusa/Medusa#10128] [Q&A]
Is there anyway to switch indexer for shows other than remove them and add them again with another indexer?

**Comment by p0psicles** [comment]
I was working on a PR for that. But I wanted to have users test it before merging. But nobody was interested.

**Reply by papampi** [reply]
Let me know how can I help

**Reply by johnmc98683** [reply]
I also would like to see this feature.

**Reply by p0psicles** [reply]
It's already added. Currently in develop branch

**Reply by papampi** [reply]
> It's already added. Currently in develop branch

I'm on dev branch, cant find how to change indexer.
Checked edit show and mass update.

Edit:
Found it under "Manage" menu
